### PR TITLE
Add change info to `dss-ops.py {lambda,secrets}`

### LINKED ISSUE #21: dss-ops lambda/secrets should print store name
The commands `dss-ops.py lambda X` and `dss-ops.py secrets X` print helpful messages that keep the user from unintentionally overwriting values. But it currently does not print the name of the secrets/param store in which the variables are being changed.

Example:

```
$ ./scripts/dss-ops.py lambda update

        *** WARNING!!! ***

        Calling the lambda update function will overwrite the current
        values of the lambda function environment stored in the
        SSM store at $DSS_DEPLOY_STAGE/environment with local
        values from environment variables on your machine.

        Note:
        - To do a dry run of this operation first, use the --dry-run flag.
        - To ignore this warning, use the --force flag.
        - To see the current environment stored in the SSM store, run:
            ./dss-ops.py lambda environment

        Are you really sure you want to update the SSM store environment?
        (Type 'y' or 'yes' to confirm):
        y
Finished redeploying local environment to lambda function environment stored in SSM store under $DSS_DEPLOY_STAGE/environment
```

Related: [chmreid-environment branch](https://github.com/DataBiosphere/data-store/tree/chmreid-environment)

### PR DESCRIPTION
**Pull request by natanlao**
Updates the warning messages used in `dss-ops.py lambda` and
`dss-ops.py secrets` to show more information about what is being
changed.

Closes #21

### CODE REVIEW

**Comment by natanlao** [comment]
I'm too scared to test this cause I don't have platform-sc access set up locally and I don't want to break dev but it seems innocuous enough

**Comment by chmreid** [comment]
@natanlao you should be able to test this locally using latest `environment` file from master, since it's only interacting with the secrets manager/param store

**Review by amarjandu — APPROVED** [review]

File: dss/operations/lambda_params.py [hunk 1]
```diff
@@ -421,7 +421,8 @@ def lambda_update(argv: typing.List[str], args: argparse.Namespace):
         - To see the current environment stored in the SSM store, run:
             ./dss-ops.py lambda environment
 
-        Are you really sure you want to update the SSM store environment?
+        Are you really sure you want to update the SSM store environment
```
Comment (chmreid) [inline dss/operations/lambda_params.py]:
```suggestion
        Are you really sure you want to update the environment in SSM store
```

File: dss/operations/secrets.py [hunk 2]
```diff
@@ -223,7 +223,8 @@ def set_secret(argv: typing.List[str], args: argparse.Namespace):
             - To do a dry run of this operation first, use the --dry-run flag.
             - To ignore this warning, use the --force flag.
 
-            Are you really sure you want to update the secret?
+            Are you really sure you want to update the secret "{secret_name}"
+            in {get_secret_store_prefix()}?
```
Comment (chmreid) [inline dss/operations/secrets.py]:
```suggestion
            in secrets store "{get_secret_store_prefix()}"?
```

File: dss/operations/lambda_params.py [hunk 3]
```diff
@@ -421,7 +421,8 @@ def lambda_update(argv: typing.List[str], args: argparse.Namespace):
...
+        {get_ssm_variable_prefix()}?
```
Comment (chmreid) [inline dss/operations/lambda_params.py]:
```suggestion
        "{get_ssm_variable_prefix()}"?
```

**Review by chmreid — APPROVED** [review]
Looks good - a few suggestions but they are nits

**Comment by natanlao** [comment]
Merging without passing CI with authorization from @amarjandu

**Pull request merged**